lib/extensions/propshaft/asset.rb

```diff
@@ -1,7 +1,13 @@
+# Monkey-patch Propshaft::Asset to produce a shorter digest as an optimization to built output size.
+#
+# See: https://github.com/rails/propshaft/blob/main/lib/propshaft/asset.rb
+
 module Extensions
   Propshaft::Asset.class_eval do
+    alias_method :original_digest, :digest
+
     def digest
-      @digest ||= Digest::SHA1.hexdigest("#{content}#{version}")[0...7]
+      @digest ||= original_digest[0...7]
```

Comment:
does the original method we're overriding also set the `@digest` ivar? if so... this may not actually override it?

Reply:
```suggestion
      @shorter_digest ||= original_digest[0...7] # rubocop:disable Naming/MemoizedInstanceVariableName
```
rubocop will yell because the ivar name doesn't match but we can override

Reply:
Hm, I thought I had tested this, but checking again, it does look like it reverted back to the original behavior with this change. Maybe it's best to just duplicate as it was in my earlier iteration.

Reply:
Fixed/reverted in 72ac3b735.